lib/node_modules/@stdlib/stats/base/dists/triangular/cdf/benchmark/benchmark.js

```diff
@@ -62,19 +72,24 @@ bench( pkg+':factory', function benchmark( b ) {
 	var mode;
 	var min;
 	var max;
+	var len;
 	var x;
 	var y;
 	var i;
 
+	len = b.iterations;
```

Comment:
```suggestion
	len = 100;
```

Throughout the PR, `len` is always initialized to 100.